documentation/metrics.md

```diff
@@ -0,0 +1,103 @@
+### Ring Middleware for HTTP Metrics
+
+The other marquee feature of the `trapperkeeper-metrics` library is a Ring
```

Comment:
"The other marquee feature" seems weird to say given that this isn't the `trapperkeeper-metrics` library?